man/src/config.1.md

```diff
@@ -0,0 +1,196 @@
+% OXIDE-CONFIG(1) oxide-config 0.1.0
+% Philipp Kalinowski
+% February 2023
+
+# NAME
+
+oxide-config - config
+
+# FILES
+
+When starting, Oxide has two paths it searches for config files.
+
+**~/.config/oxide/config.yml**
+: Home config file
+
+**/etc/oxide/config.yml**
+: System config file
+
+# Description
+
+Define the behaviour of oxidewm. You can change style, layout, keybindings and more.
+The config file is written using YAML syntax.
+If the home config file is not existing, default values will be used but commands like `exec` and `exec_always` will not be working.
+
+# KEYBINDING
+
+## KEYS
+
+The keys need at least one MODIFIER and one normal key such as 't'
+
+### MODIFIER
+
+**M**
+: Meta key
+
+**A**
+: ALT key
+
+**C**
+: CONTROL key
+
+**S**
+: SHIFT key
+
+## COMMANDS
+
+Commands consist of a command and optional arguments.
+
+### COMMAND
```

Comment:
Man pages have only two levels of headlines # and ## Please work with that accordingly

I would propose to put your little introduction just under one headline and then list the command below it